src/app/modules/bounce/bounce.service.ts

```diff
@@ -266,14 +266,22 @@ export const logEmailNotification = (
   // form fillers' information for too long, and everything else goes into the
   // main log group.
   const emailType = extractHeader(notification, EMAIL_HEADERS.emailType)
+  const isBounce = isBounceNotification(notification)
+  const message = isBounce ? 'Email bounced' : 'Email delivered'
   if (
     emailType === EmailType.EmailConfirmation ||
     emailType === EmailType.VerificationOtp
   ) {
-    shortTermLogger.info(notification)
+    shortTermLogger.info({
```

Comment:
Update the logger info body to be consistent - so it is easy to find the logs.

Reply:
ie, for shortTermLogger, it logs the whole notification object, making it hard to search. It differs in the long term logger which has a `meta.action` for easier log search.